lib/global_uid/base.rb

```diff
@@ -76,17 +76,13 @@ def self.init_server_info(options)
       raise "You haven't configured any id servers" if id_servers.nil? or id_servers.empty?
       raise "More servers configured than increment_by: #{id_servers.size} > #{options[:increment_by]} -- this will create duplicate IDs." if id_servers.size > options[:increment_by]
 
-      offset = 1
-
       id_servers.map do |name, i|
         info = {}
         info[:cx]       = nil
         info[:name]     = name
         info[:retry_at] = nil
-        info[:offset]   = offset
         info[:rand]     = rand
         info[:new?]     = true
-        offset +=1
```

Comment:
You'd be right to feel uneasy when seeing the removal of this `offset`, the gem works by incrementing with the `AUTO_INCREMENT_INCREMENT` and `AUTO_INCREMENT_OFFSET` after-all. 

However, after a lot of checking and re-checking I can confirm that this offset was introduced in the first commit 2744dff891091053adfecb42718c6126c52acf78 and *only ever* used by the `GlobalUid::ServerVariables` module. 

The option could have been cleaned up as part of 74746849d6c72d2c1969a8c5f93b8346388727ad, but was missed.

The alloc server is expected to have the `auto_increment_offset` configured globally in production environments.

Reply:
Also, for what it's worth, relying on it being set within the gem would lead to issues. The variable used to be set [per session](https://github.com/zendesk/global_uid/pull/66/files#diff-09178000d0615af7e621a2fe995b9f42L23) and if the order of `id_servers` changed, the offset would be configured differently between sessions/connections 😱